### PR TITLE
Reworked proxy-port and added flag for monitor-port

### PR DESCRIPTION
**Pull request by ineiti**
- Changed the occurences of ```Logger``` to ```Monitor```
- Added monitor_port to flags in ```deploy```
- Removed ProxyAddressPort and replaced with monitor_port + 1

### CODE REVIEW

**Comment by nikkolasg** [comment]
We do not necessarily need the Monitor + MonitorPort flags on App since apps will only be using the full address to connect. That way you don't have to change every apps. 
We just need to make sure localhost.go + forkexec.go gives the rights address to apps.

**Comment by nikkolasg** [comment]
Forkexec / Users need to give directly the full address now.

**Comment by ineiti** [comment]
OK to merge.

File: deploy/platform/deterlab/users/users.go [hunk 1]
```diff
@@ -124,11 +123,14 @@ func main() {
 
 	// ADDITIONS : the monitoring part
 	// Proxy will listen on Sink:SinkPort and redirect every packet to
-	// RedirectionAddress:RedirectionPort. With remote tunnel forwarding it will
+	// RedirectionAddress:SinkPort-1. With remote tunnel forwarding it will
```
Comment (nikkolasg) [inline deploy/platform/deterlab/users/users.go]: RedirectionAddress:SinkPort + 1 , no ?

**Comment by nikkolasg** [comment]
Beside my comment, OK to merge for me.

**Comment by Liamsi** [comment]
I also think it is ready to merge.

**Pull request merged**